client-cli/src/command/transaction_command.rs

```diff
@@ -157,6 +208,34 @@ impl TransactionCommand {
                 success(format!("import amount: {}", imported_amount).as_str());
                 Ok(())
             }
+            TransactionCommand::Build { name } => {
+                let enckey = ask_seckey(None)?;
+                let to_address = ask_transfer_address()?;
+                ask("Enter transfer amount (in CRO): ");
```

Comment:
make sure that user enters  "." in cro

Reply:
yes, the function [`coin_from_str`](https://github.com/crypto-com/chain/blob/master/client-cli/src/main.rs#L75) can deal with that I think.